README.md

```diff
@@ -1,9 +1,59 @@
-# belgrade-weather
+Here’s the `README.md` file in markdown format:
+# Weather Data Logger
 
-This repo contains historical data of Belgrade weather. To take a look, you can
-either directly open `history.csv`, or open a
-[Flat Data representation](https://flatgithub.com/102/belgrade-weather).
+## Description
+**Weather Data Logger** is a Deno-based application that captures and logs weather data from a JSON file into a CSV format. This allows users to maintain a historical record of weather conditions, making it easy to analyze trends over time. 
+
+This project is designed as a lightweight solution for personal or small-scale weather tracking and is not intended for serious research purposes.
+
+## Features
+- **Data Logging**: Reads weather data from a JSON file and appends it to a CSV file.
```

Comment:
this doesn't really add any additional information as data is not just read from json, it is being logged from the api